docs/en/sql-reference/statements/select/order-by.md

```diff
@@ -221,3 +221,70 @@ returns
 │ 1970-03-12 │ 1970-01-08 │ original │
 └────────────┴────────────┴──────────┘
 ```
+
+## OFFSET FETCH Clause {#offset-fetch}
+
+`OFFSET` and `FETCH` allow you to retrieve just a portion of the rows that are generated by the rest of the query.
```

Comment:
1. There is no "rest of the query" after this clause. OFFSET FETCH is a final part of a query that limits the number of rows.
2. What is the motivation or a usecase for a user to use this clause?